src/System.Linq/src/System/Linq/First.cs

```diff
@@ -34,17 +32,11 @@ public static TSource First<TSource>(this IEnumerable<TSource> source, Func<TSou
             return first;
         }
 
-        public static TSource FirstOrDefault<TSource>(this IEnumerable<TSource> source)
-        {
-            bool found;
-            return source.TryGetFirst(out found);
-        }
+        public static TSource FirstOrDefault<TSource>(this IEnumerable<TSource> source) =>
```

Comment:
I agree with @sharwell on this, but have been following @stephentoub on it after an earlier discussion on the same point.